### PR TITLE
added bootstrap and basic styling

### PR DESCRIPTION
**Pull request by Sajaas**
## PT: https://www.pivotaltracker.com/story/show/152763280
* Added and installed bootstrap
* Added container, row and columns
* Resized the product image, made it round
screen shot 2017-11-10 at 20 19 00

### CODE REVIEW

**Comment by olofvi** [comment]
Could you add a screenshot of the changes to the PR ;)

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  <h1>{{product.price}}</h1>
-  <h1>{{product.description}}</h1>
-  <img src={{product.image}} />
-  <h1>{{product.type}}</h1>
-  <h1>{{product.status}}</h1>
+<div class="container">
+  <div class="row">
+    <div class="col-md-4 centered" *ngFor="let product of products">
+      <h1>{{product.name}} {{product.price}}:-</h1>
+      <!--<h1>{{product.description}}</h1>-->
```
Comment (olofvi) [inline src/app/app.component.html:5]: why is there commented out code here?

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,10 +1,11 @@
-<div *ngFor="let product of products">
-  <h1>{{product.name}}</h1>
-  <h1>{{product.price}}</h1>
-  <h1>{{product.description}}</h1>
-  <img src={{product.image}} />
-  <h1>{{product.type}}</h1>
-  <h1>{{product.status}}</h1>
+<div class="container">
```
Comment (olofvi) [inline src/app/app.component.html:1]: I think you should take a look at using the ng-container here

File: src/app/app.component.html (hunk 1 above)
Comment (Sajaas) [reply src/app/app.component.html:5]: I was going to remove it! :)

File: src/app/app.component.html (hunk 2 above)
Comment (Sajaas) [reply src/app/app.component.html:1]: But, but, but I liiike the Bootstrap containers! 😄

Comment (olofvi) [reply src/app/app.component.html:1]: hmm yeah but I think you could do ```<ng-container class:container></ng-container>``` for data-binding the html but maybe thats out of scope for basic styling ;)

**Comment by olofvi** [comment]
forget about the ng-container this is a great start to work with I'll merge :)

**Pull request merged**